tools/src/main/scala/scala/scalanative/codegen/AbstractCodeGen.scala

```diff
@@ -84,6 +130,14 @@ private[codegen] abstract class AbstractCodeGen(
     defns.foreach { defn => if (defn.isInstanceOf[Defn.Var]) onDefn(defn) }
     defns.foreach { defn => if (defn.isInstanceOf[Defn.Declare]) onDefn(defn) }
     defns.foreach { defn => if (defn.isInstanceOf[Defn.Define]) onDefn(defn) }
+
+    // defns.foreach {
```

Comment:
While I guess the original block of code separates variables from declarations, from defines, I wonder if it'd be faster to do a single pass over this collection.

It's likely to be insignificant though.